issue-artifacts/941/runtime-architecture-review-2026-08-07.html

```
@@ -0,0 +1,1218 @@
+					<div class="mb-3 flex items-center justify-between gap-4">
+						<h2 class="text-xl font-black text-white">
+							Recommended target shape
+						</h2>
+						<span
+							class="rounded-full bg-cyan/10 px-3 py-1 text-xs font-semibold text-cyan"
+							>arrows = consumption</span
+						>
+					</div>
+					<pre class="mermaid">
+flowchart LR
+  HEVY["Hevy HTTP"] --> CLIENT["hevy-client\nrequest adapter"]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)